src/types/filters/index.d.ts

```diff
@@ -31,6 +31,7 @@ export type KeywordState = Array<{
 	avatarBackground?: ChipProps['background'];
 	hasError?: boolean;
 	error?: boolean;
+	fullName?: string;
```

Comment:
The _KeywordState_ type is  a multipurpose type used for every ChipInput in the search UI (despite its misleading name).
IMO adding the _fullName_ property it is not a proper solution because is adding a Contact specific data type to a generic one